native/gulpfile.js

```diff
@@ -147,11 +147,10 @@ gulp.task('clean-simulator', async function () {
     let isWin32 = process.platform === 'win32';
     let delPatterns = [
         formatPath(Path.join(__dirname, './simulator/*')),
-        formatPath(`!${Path.join(__dirname, './simulator/Debug')}`),
+        formatPath(`!${Path.join(__dirname, './simulator/Release')}`),
     ];
     if (!isWin32) {
-        delPatterns.push(formatPath(Path.join(__dirname, './simulator/Debug/libcocos2d.a')));
```

Comment:
we don't generate libcocos2d.a any more ?

Reply:
Yes.